File: DP_FamilyPhotoSharing-main/DP_FamilyPhotoSharing-main/ModelLayer/PhotoEnums/ThumbnailDataEnum.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ModelLayer.PhotoEnums
{
    public enum ThumbnailData : int
    {
        [Description("Soukromá galerie")]
        MainPersonalGallery = 1,
        [Description("Rodinná galerie")]
        MainGroupGallery = 2,
        [Description("Soukromé album")]
        Album = 3,
        [Description("Rodinné album")]
        GroupAlbum = 4,
        [Description("Vlastní sdílené album")]
        MySharedAlbum = 5,
        [Description("Sdílené album")]
        SharedAlbum = 6,
        [Description("Rodinná galerie - vlastní foto")]
        MainGroupGalleryOwnPhoto = 7,
    }
}

File: DP_FamilyPhotoSharingiOS-main/DP_FamilyPhotoSharingiOS-main/DP_iOSApp/FamilyPhotoSharing/FamilyPhotoSharing/Enums/ThumbnailDataEnum.swift
//
//  ThumbnailDataEnum.swift
//  FamilyPhotoSharing
//
//  Created by Jiří Gába on 01.05.2026.
//

enum ThumbnailData: Int, Codable {
    case mainPersonalGallery = 1
    case mainGroupGallery = 2
    case album = 3
    case groupAlbum = 4
    case mySharedAlbum = 5
    case sharedAlbum = 6
    case mainGroupGalleryOwnPhoto = 7

    var description: String {
        switch self {
        case .mainPersonalGallery:
            return "Soukromá galerie"
        case .mainGroupGallery:
            return "Rodinná galerie"
        case .album:
            return "Soukromé album"
        case .groupAlbum:
            return "Rodinné album"
        case .mySharedAlbum:
            return "Vlastní sdílené album"
        case .sharedAlbum:
            return "Sdílené album"
        case .mainGroupGalleryOwnPhoto:
            return "Rodinná galerie - vlastní foto"
        }
    }
}
